### PR TITLE
MariaDB

### PR DESCRIPTION
**Pull request by jakolehm**

### CODE REVIEW

File: mariadb/kontena.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    type: string
+    from:
+      evaluate: ((${mem_limit} * 10) * 7) / 100
+services:
+  server:
+    image: mariadb:10.2
+    instances: 1
+    command: "mysqld --innodb-buffer-pool-size={{ innodb_buffer_pool_size }}M"
+    mem_limit: "{{ mem_limit }}m"
+    stateful: true
```
Comment (jnummelin) [inline mariadb/kontena.yml:56]: Why both `stateful: true` and volumes for the data?

File: mariadb/kontena.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      prompt: Affinity
+  mem_limit:
+    type: integer
+    default: 512
+    from:
+      prompt: Memory limit (MB)
+  innodb_buffer_pool_size:
+    type: string
+    from:
+      evaluate: ((${mem_limit} * 10) * 7) / 100
```
Comment (jnummelin) [inline mariadb/kontena.yml:49]: Why such a complex formula? Is there some reason why `${mem_limit} * 0,7` would not work?

**Review by jnummelin — COMMENTED** [review]
Having the DB both stateful and with volumes looks bit wrong

File: mariadb/kontena.yml (hunk 2 above)
Comment (jakolehm) [reply mariadb/kontena.yml:49]: It needs to be an integer.

File: mariadb/kontena.yml (hunk 1 above)
Comment (jakolehm) [reply mariadb/kontena.yml:56]: Stateful affects also how scheduler behaves. Of course it can be removed but it's not just for volumes here.

**Pull request merged**